frontend/src/metabase/redux/downloads.ts

```diff
@@ -206,20 +206,23 @@ const getDatasetParams = ({
       return {
         method: "GET",
         url: `/api/embed/dashboard/${token}/dashcard/${dashcardId}/card/${cardId}/${type}`,
-        params: Urls.getEncodedUrlSearchParams({ ...params, ...exportParams }),
+        params: new URLSearchParams({
+          parameters: JSON.stringify(params),
+          ..._.mapObject(exportParams, value => String(value)),
+        }),
       };
     }
 
     if (resource === "question" && token) {
-      // For whatever wacky reason the /api/embed endpoint expect params like ?key=value instead
-      // of like ?params=<json-encoded-params-array> like the other endpoints do.
```

Comment:
We also removed this comment in the [original fix](https://github.com/metabase/metabase/pull/46250/files#diff-7a0c448475100a29b2d3e28699cf20381b32b5b785fd6e4128966e1620cb3987)